From 1a62bdf4c4864ee3cc2c2e971e50ec77b77129fd Mon Sep 17 00:00:00 2001
From: Luca Weiss <luca@lucaweiss.eu>
Date: Sun, 7 Sep 2025 16:47:33 +0200
Subject: [PATCH] Move logread binary to sbin

Move logread to the same location as busybox logread to not install them
side-by-side in Alpine Linux.
---
 meson.build | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/meson.build b/meson.build
index 7bfd0d3..431b2ec 100644
--- a/meson.build
+++ b/meson.build
@@ -20,7 +20,8 @@ executable('logbookd', 'main.c', 'shared.h', 'shared.c', 'parser.c', 'parser.h',
     install: true)
 
 executable('logread', 'logread.c', 'shared.h', 'shared.c', 'bytemess.h',
-    install: true)
+    install: true,
+    install_dir: get_option('sbindir'))
 
 
 scdoc = dependency('scdoc', required: get_option('man-pages'), version: '>= 1.9.7', native: true)
-- 
2.51.0

